### ISSUE TITLE
Improve the error message when old-style SonarLint.xml file is present

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonarlint-visualstudio#1519]
The current error message is confusing, as it points users to delete the SonarLint.xml file from disk, but it should also mention to remove the SonarLint.xml file from any `csproj` or `build.props` files, as well. 

Repro steps:
1. clone sonar-security (it has the old SL4VS config in source control)
2. `git checkout 8.4.0.2759`
3. go to Team Explorer -> SonarQube , right click on our SQ project and click Update

Behaviour (SonarLint Output):
```
Binding solution to SonarQube project: Download quality profile(s):
   Successfully downloaded quality profile. Name: 'C# QP for SonarSource', Key: 'cs-c-qp-for-sonarsource-73109', Language: 'C#'
INFO: SonarLint connected mode no longer installs the analyzers NuGet packages. The analyzers embedded in the VSIX are used instead.
Binding solution to SonarQube project: Generating project rule sets
Ruleset binding: all projects need to be updated reference the generated ruleset.
Unexpected error during workflow execution: Failed to bind project 'C:\<REDACTED>\<PROJECTNAME>\<PROJECTNAME>.csproj'.
A conflicting version of SonarLint.xml has been found. Please remove 'C:\<REDACTED>\SonarLint.xml', re-open the solution and try again.
```

4. go to 'C:\<REDACTED>` and delete the SonarLint.xml file
5. close Visual Studio and reopen the solution
6. repeat step 3 to update the QP 

Behaviour (SonarLint Output): the same error, saying `Please remove 'C:\<REDACTED>\SonarLint.xml', re-open the solution and try again.`, even if the SonarLint.xml file isn't present there anymore

Expected behaviour: an error message that also says to remove the reference of the old sonarlint.xml file from the project file / in a Directory.Build.props file. Maybe a link to [the docs on the wiki](https://github.com/SonarSource/sonarlint-visualstudio/wiki/Connected-Mode-configuration-for--C%23-and-Visual-Basic-projects#can-i-customize-how-my-projects-are-configured)  (see [internal discussion](https://discuss.sonarsource.com/t/add-link-to-slvs-documentation-on-sonarlint-org/5009/4)).

Environment:
- Windows 10
- Visual Studio 2019 16.5.4
- SonarLint 4.22.0.18519
